src/FOHClasses/PhysicalTable.java

```diff
@@ -12,7 +12,8 @@ public class PhysicalTable {
     public PhysicalTable(int tableID, int childSeats) {
         this.tableID = tableID;
         this.childSeats = childSeats;
-        //TODO add this.assignedWaiters and this.tableCollection
+        this.assignedWaiters = new WaiterCollection();
+        this.tableCollection = new TableCollection();
```

Comment:
A PhysicalTable would not contain any other tables, as a PhysicalTable is at the bottom of the hierarchy/tree. Something that could be implemented however is a pointer that points to the BookableTable object that this PhysicalTable is a part of.